src/NexusMods.Games.CreationEngine/ACreationEngineSynchronizer.cs

```diff
@@ -0,0 +1,18 @@
+using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.Loadouts.Synchronizers;
+using NexusMods.Paths;
+
+namespace NexusMods.Games.CreationEngine;
+
+public abstract class ACreationEngineSynchronizer : ALoadoutSynchronizer
+{
+    protected ACreationEngineSynchronizer(IServiceProvider provider) : base(provider)
+    {
+    }
+
+    public override bool IsIgnoredBackupPath(GamePath path)
+    {
+        // Don't backup BSA files by default
+        return path.Extension == KnownCEExtensions.BSA || path.Extension == KnownCEExtensions.BA2;
+    }
```

Comment:
If I understand things correctly, this should no longer be needed in the future, as the synchronizer should be changed to no longer backup files except if they are to be replaced by a mod or things like that?